OrbitClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientModel/SamplingDataPostProcessor.h"
+
+#include <cstdint>
+#include <cstring>
+#include <map>
+#include <memory>
+#include <set>
+#include <vector>
+
+#include "OrbitClientData/Callstack.h"
+#include "OrbitClientData/CallstackTypes.h"
+#include "OrbitClientData/PostProcessedSamplingData.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::CallstackEvent;
+using orbit_client_protos::FunctionInfo;
+using orbit_client_protos::LinuxAddressInfo;
+
+namespace sampling_data_post_processor {
+
+namespace {
+
+class SamplingDataPostProcessor {
```

Comment:
IMO it would be a bit more readable if class declaration is split from the implementation (like in ProcessManagerImpl).

Reply:
Should be done

Reply:
I haven't meant to split on the separate namespaces though 🙃

Reply:
I haven't meant to split on the separate namespaces though 🙃